databases/week3/Meal sharing.sql

```diff
@@ -0,0 +1,211 @@
+-- Create all the sql for creating the data model
+
+set names utf8mb4;
+CREATE DATABASE MealSharing;
+
+CREATE TABLE meal (
+id int unsigned NOT NULL AUTO_INCREMENT,
+title varchar(255) NOT NULL,
+description TEXT,
+location varchar(255) NOT NULL,
+`when` DATETIME NOT NULL,
+max_reservations INT,
+price decimal NOT NULL,
+created_date DATE NOT NULL,
+PRIMARY KEY (id)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+CREATE TABLE reservation (
+id INT unsigned NOT NULL AUTO_INCREMENT,
+number_of_guests INT unsigned NOT NULL DEFAULT 0,
+meal_id INT unsigned NOT NULL,
+created_date DATE NOT NULL,
+contact_phonenumber VARCHAR(20) NOT NULL,
+contact_name VARCHAR(200) NOT NULL,
+contact_email VARCHAR(100),
+PRIMARY KEY (id),
+CONSTRAINT `fk_meal` FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
+);
```

Comment:
I think you forgot to add `ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci` to this table